spec/helpers/step_nav_actions_helper_spec.rb

```diff
@@ -116,9 +116,9 @@
         expect(helper.must_check_for_broken_links?(step_by_step_page)).to be true
       end
 
-      it "returns false if link checker has been run and there are no broken links" do
+      it "returns false if link checker has been run and there are broken links" do
```

Comment:
I'd maybe rephrase this to:

> `it "returns false even if the latest link report found broken links"`

Current wording sounds like it should return false _because_ it found broken links, but really the presence of broken links is inconsequential.